terraform_modules/base_framework/bastion.tf

```diff
@@ -0,0 +1,115 @@
+data aws_ami ubuntu {
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-jammy-*-amd64-server-*"]
```

Comment:
Didn't find any way to point to the latest ubuntu release. The `most_recent = true` always points to the latest version within a specific release and not to the latest across releases. I added a variable so we can easily set which release we want to use.

Reply:
I was not sure if it was possible or not, just asked. If you can set it as a variable or anything it would be awesome. But it is OK as it is and do it in a second iteration.